src/crimson/osd/pg.h

```diff
@@ -159,6 +159,16 @@ class PG : public boost::intrusive_ref_counter<
     bool need_write_epoch,
     ceph::os::Transaction &t) final;
 
```

Comment:
I'm confused by this commit.  ceph/main already has a commit with this sha1 at 948e8ee394c374332b3043de09f5b67078a86418 and the commit message doesn't appear related to the contents here.